<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="group_Message">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>411</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Message Box</string>
   </property>
   <widget class="QTextBrowser" name="MessageBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>391</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="group_B">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>781</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Measured Magnetic Field</string>
   </property>
   <widget class="QPushButton" name="StopInput_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>265</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Input</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkX_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">X-axis</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkZ_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Z-axis</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkY_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Y-axis</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="GraphicsLayoutWidget" name="MeasuredB_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>761</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="StopInput_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>265</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Measure</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkflux">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>/   Fluxgate</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="group_Wave">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>371</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Expected Wave Form</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="group_Settings">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>411</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Settings</string>
   </property>
   <widget class="QLabel" name="Amp_l_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Amplitude</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Amp_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="Freq_l_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Freq_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="Dur_l_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Duration</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Dur_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <widget class="QLabel" name="Keep_l_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Keep Time</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Keep_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="Offset_l_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Offset</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Offset_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Dev_l_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Device</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Dev_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="Port_l_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Port_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FormUpdate_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Update </string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>301</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>85</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
  </widget>
  <widget class="GraphicsLayoutWidget" name="waveform">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>351</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>Amp_2</tabstop>
  <tabstop>Keep_2</tabstop>
  <tabstop>Dev_2</tabstop>
  <tabstop>Freq_2</tabstop>
  <tabstop>Offset_2</tabstop>
  <tabstop>Port_2</tabstop>
  <tabstop>Dur_2</tabstop>
  <tabstop>FormUpdate_2</tabstop>
  <tabstop>StartButton_2</tabstop>
  <tabstop>checkZ_2</tabstop>
  <tabstop>StopInput_2</tabstop>
  <tabstop>waveform</tabstop>
  <tabstop>checkX_2</tabstop>
  <tabstop>MessageBox_2</tabstop>
  <tabstop>MeasuredB_2</tabstop>
  <tabstop>checkY_2</tabstop>
  <tabstop>StopInput_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
